features/all/phpcr_query_update.feature

```diff
@@ -153,3 +153,30 @@ Feature: Execute a a raw UPDATE query in JCR_SQL2
             | UPDATE [nt:unstructured] mixin_foo('bar') |
             | UPDATE [nt:unstructured] APPLY mixin_foo('bar') |
             | UPDATE [nt:unstructured] mixin_foo'bar') |
+
+    Scenario Outline: Execute update query with expressions
+        Given I execute the "<query>" command
```

Comment:
this looks like a WHEN step, not a GIVEN one, when reading it: it describes the action being done, not the initial state.